src/Analyser/TypeSpecifier.php

```diff
@@ -1191,11 +1181,6 @@ private function specifyTypesFromAsserts(TypeSpecifierContext $context, Expr\Cal
 			return null;
 		}
 
-		$originalAsserts = null;
-		if ($parametersAcceptor instanceof ResolvedFunctionVariant) {
-			$originalAsserts = Assertions::fromParametersAcceptor($parametersAcceptor->getOriginalParametersAcceptor())->getAll();
```

Comment:
/cc @rvanvelzen It's interesting that the `$containsUnresolvedTemplate` removal doesn't cause any tests to fail. Any idea what I could have broken with this?

Reply:
There wasn't yet a test for this, but I included it now. It was one of the major errors in the rector integration test. There's a different one that I'm investigating now, but should be ready soon.